plugins/get_url/komand_get_url/triggers/poll_file/trigger.py

```diff
@@ -14,65 +15,48 @@ def __init__(self):
             input=PollFileInput(),
             output=PollFileOutput(),
         )
+        self.is_modified = True
+        self.utils = Utils(action=self)
 
     def run(self, params={}):
-        utils = Utils(action=self)
-        url = params.get(Input.URL)
-        is_verify = params.get(Input.IS_VERIFY, True)
         poll = params.get(Input.POLL, 60)
 
         while True:
-            """Check for supported url prefix"""
-            utils.validate_url(url)
-
-            is_modified = True
-            meta = utils.hash_url(url)
-            cache_file = "/var/cache/" + meta["file"]
-
-            """Attempt to retrieve headers from past request"""
-            headers = {}
-            if komand.helper.check_cachefile(meta["metafile"]):
-                headers = utils.check_url_meta_file(meta)
-
-            """Download file"""
-            url_object = komand.helper.open_url(
-                url,
-                verify=is_verify,
-                If_None_Match=headers.get("etag", ""),
-                If_Modified_Since=headers.get("last-modified", ""),
-                User_Agent=params.get(Input.USER_AGENT, "Mozilla/5.0"),
-            )
-
+            url = params.get(Input.URL)
+            is_verify = params.get(Input.IS_VERIFY, True)
```

Comment:
Why are these inside the loop?

Reply:
There is little harm in getting the user input values on every loop, but there is also little value because the values do not change between loops. Values would only change when the entire trigger is restarted and forcibly breaking this infinite loop.
It would be an improvement to move these variable retrievals before the infinite loop, but a minor one.